Localizations/en_US.json

```diff
@@ -148,13 +148,13 @@
     "script_content_level_1": "Level 1",
     "script_content_level_1_description": "Methods are stripped from decompiled export.",
     "script_content_level_2": "Level 2",
-    "script_content_level_2_description": "Default. This exports full methods for Mono games and dummy methods for IL2Cpp games.",
+    "script_content_level_2_description": "Default. This exports full methods for Mono games and dummy methods for IL2CPP games.",
     "script_content_level_3": "Level 3",
-    "script_content_level_3_description": "IL2Cpp methods are safely recovered where possible.",
+    "script_content_level_3_description": "IL2CPP methods are safely recovered where possible.",
     "script_content_level_4": "Level 4",
-    "script_content_level_4_description": "IL2Cpp methods are recovered without regard to safety.",
+    "script_content_level_4_description": "IL2CPP methods are recovered without regard to safety.",
```

Comment:
Please change it back. I format it that way because IL is a two-letter acronym, and Cpp is a three-letter acronym.

Reply:
Sure! I think I forgot about those. 😅